client/CMT.cpp

```diff
@@ -864,7 +865,7 @@ static bool recreateWindow(int w, int h, int bpp, bool fullscreen)
 
 	int suggestedBpp = bpp;
 
-	if(!checkVideoMode(0,w,h,suggestedBpp,fullscreen))
+	if(!checkVideoMode(0, w, h, suggestedBpp, fullscreen))

```

Comment:
Should the code check the video mode for all the available displays (you can get a number of available displays with the `SDL_GetNumVideoDisplays` function)? But this is probably messy a bit, dunno. Is there a way to tell SDL on which display it should show the created window?